# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/vm/GlobalObject.cpp
# Commit: 3b62bd57747f
# Full Hash: 3b62bd57747f21c800a93da325d999fa5efb3850
# Author: Jessica Tallon <jtallon@igalia.com>
# Date: 2021-09-03 03:51:58
# Regressor Bug: 1715275
# File Overlap Count: 1
# Description:
#   Bug 1715275 - Add support for a JS WebAssembly.Function object r=rhunt
#   
#   This introduces a new Constructor WebAssembly.Function which is part of
#   the WASM JS-API type reflection work. This allows normal JS functions to
#   be wrapped with a type and used with WebAssembly objects (e.g. Tables).
# ==============================================================================

diff -r f6f4514d09bf -r 3b62bd57747f js/src/vm/GlobalObject.cpp
--- a/js/src/vm/GlobalObject.cpp	Thu Sep 02 15:56:01 2021 +0000
+++ b/js/src/vm/GlobalObject.cpp	Thu Sep 02 15:58:18 2021 +0000
@@ -162,6 +162,7 @@
     case JSProto_WasmTable:
     case JSProto_WasmGlobal:
     case JSProto_WasmTag:
+    case JSProto_WasmFunction:
     case JSProto_WasmException:
       return false;
 